Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PowerBIEmbedded/PowerBIEmbeddedPage.xaml.cs

```diff
@@ -3,18 +3,80 @@
 // See the LICENSE file in the project root for more information.
 
 using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
 
 namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
 {
-    public sealed partial class PowerBIEmbeddedPage : IXamlRenderListener
+    public sealed partial class PowerBIEmbeddedPage
     {
         public PowerBIEmbeddedPage()
         {
             this.InitializeComponent();
         }
 
-        public void OnXamlRendered(FrameworkElement control)
+        private void PropertyDropdown_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
+            PropertyValue.Text = string.Empty;
+
+            switch (PropertyDropdown.SelectedIndex)
+            {
+                case 0:
+                    if (PropertyTipsGroupId != null)
+                    {
+                        PropertyTipsGroupId.Visibility = Visibility.Visible;
+                        PropertyTipsEmbedUrl.Visibility = Visibility.Collapsed;
+                    }
+
+                    break;
+                case 1:
+                    if (PropertyTipsGroupId != null)
+                    {
+                        PropertyTipsGroupId.Visibility = Visibility.Collapsed;
+                        PropertyTipsEmbedUrl.Visibility = Visibility.Visible;
+                    }
+
+                    break;
+            }
+        }
+
+        private void ClientIdExpandButton_Click(object sender, RoutedEventArgs e)
+        {
+            SetVisibilityStatusPanel(ClientIdBox, (Button)sender);
+        }
+
+        private void SetVisibilityStatusPanel(FrameworkElement box, Button switchButton)
+        {
+            if (box.Visibility == Visibility.Visible)
+            {
+                switchButton.Content = "��";
```

Comment:
Think there's an issue with these characters?

![image](https://user-images.githubusercontent.com/24302614/42787467-7d73bbbc-890f-11e8-8678-14b08d5defa6.png)

Reply:
@michael-hawker Fixed by changing encoding from ANSI to UTF-8